rust/signed_doc/src/doc_types/mod.rs

```diff
@@ -21,24 +21,12 @@ pub const REVIEW_DOCUMENT_UUID_TYPE: Uuid =
 /// Review template `UuidV4` type.
 pub const REVIEW_TEMPLATE_UUID_TYPE: Uuid =
     Uuid::from_u128(0xEBE5_D0BF_5D86_4577_AF4D_008F_DDBE_2EDC);
-/// Category document `UuidV4` type.
-pub const CATEGORY_DOCUMENT_UUID_TYPE: Uuid =
```

Comment:
These document types still exist.  They have not been removed.
See: https://input-output-hk.github.io/catalyst-libs/architecture/08_concepts/signed_doc/docs/brand_parameters/
https://input-output-hk.github.io/catalyst-libs/architecture/08_concepts/signed_doc/docs/campaign_parameters/
https://input-output-hk.github.io/catalyst-libs/architecture/08_concepts/signed_doc/docs/category_parameters/

All that happened was the 3 references were consolidated into a single reference field called "parameters" there is no distinct "Parameters" document.   There are multiple parameters type documents.